src/components/Settings/SettingsMain/index.tsx

```diff
@@ -44,6 +44,9 @@ const messages = defineMessages('components.Settings.SettingsMain', {
   csrfProtectionTip: 'Set external API access to read-only (requires HTTPS)',
   csrfProtectionHoverTip:
     'Do NOT enable this setting unless you understand what you are doing!',
+  cspFrameAncestorDomains: 'Frame-Ancestor Domains',
+  cspFrameAncestorDomainsTip:
+    'Domains to allow embedding Jellyseer as iframe, object or embed. Incompatible with CSRF-Protection',
```

Comment:
Shouldn't you specify here that this is a space-separated list of domains?

Reply:
There are more possibilities to setting this than just space separated as this is directly put into the header.
But maybe we could link to the specification example: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors#examples

Reply:
Oh ok. That's a good idea yes